examples/basic/app/routes/demos/actions.tsx

```diff
@@ -19,19 +17,11 @@ export let action: ActionFunction = async ({ request }) => {
   // over the network. Clientside validation is fine, but you definitely need it
   // server side.  If there's a problem, return the the data and the component
   // can render it.
-  if (typeof answer !== "string") {
+  if (!answer || typeof answer !== "string") {
     return json("Come on, at least try!", { status: 400 });
   }
 
-  let rightAnswers = [
-    "4fa6024f12494d3a99d8bda9b7a55f7d140f328a",
-    "ce3659ad235ca6d1e12dec21465aff3f9a62bb8c",
-    "bd111dcb4b343de4ec0a79d2d5ec55a3919c79c4"
-  ];
-
-  let encrypted = hash(answer);
-
-  if (!rightAnswers.includes(encrypted)) {
```

Comment:
I have a feeling this difference from the template was intentional. I'll go ahead and restore this.

Reply:
These were deleted from the templates by @ryanflorence in #462, so that's why I deleted it here too.

I'll create a separate PR to delete it

Reply:
@kentcdodds Here you go: https://github.com/remix-run/remix/pull/779